SnowflakeTableJournal.cs

```diff
@@ -47,15 +34,20 @@ protected override string GetJournalEntriesSql()
 
         /// <summary>Verify, using database-specific queries, if the table exists in the database.</summary>
         /// <returns>1 if table exists, 0 otherwise</returns>
-        protected override string DoesTableExistSql() => string.IsNullOrEmpty(SchemaTableSchema)
-                ? $"select count(*) from INFORMATION_SCHEMA.TABLES where TABLE_NAME = '{UnquotedSchemaTableName.ToUpper()}'"
-                : $"select count(*) from INFORMATION_SCHEMA.TABLES where TABLE_NAME = '{UnquotedSchemaTableName.ToUpper()}' and TABLE_SCHEMA = '{SchemaTableSchema.ToUpper()}'";
+        protected override string DoesTableExistSql()
+        {
+            var sql = $"select count(*) from INFORMATION_SCHEMA.TABLES where TABLE_NAME = '{UnquotedSchemaTableName.ToUpper()}'";
+            if (!string.IsNullOrEmpty(SchemaTableSchema))
+                sql += $" and TABLE_SCHEMA = '{SchemaTableSchema.ToUpper()}'";
+
+            return sql;
+        }
 
         protected override string CreateSchemaTableSql(string quotedPrimaryKeyName)
         {
             return $@"CREATE TABLE {FqUnquotedSchemaTableName}
             (
-                schemaversionsid int identity,
+                schemaversionsid int identity constraint {UnquoteSqlObjectName(quotedPrimaryKeyName)} primary key,
```

Comment:
Why are we unquoting identifiers here again? (I'm just blindly following suit with the table name.)